lib/SparkPost/Transmission.php

```diff
@@ -63,12 +63,22 @@ private function __construct() {}
 	 */
 	private static function getHttpClient() {
 		if(!isset(self::$request)) {
-			self::$request = new Client();
+			$hostConfig = SparkPost::getConfig();
+			$defaultOptions = [
+				'base_uri'=>self::getBaseUrl($hostConfig),
+				'headers' =>[
+					'Authorization' => $hostConfig['key'],
+					'Content-Type' => 'application/json',
+					'User-Agent' => 'SparkPost PHP SDK v0.1.2'
```

Comment:
@richleland - I added this in.  What do you think / should we have it?  Also we need to decide what the release version should be since this is a pretty large update that will be breaking changes for people using PHP 5.3 or PHP 5.4.